README.md

```diff
@@ -11,7 +11,9 @@ This repositorry contain a method to compare metagenomics and metabarcoding.
 
 Remaining tasks, clarifications, or questions:
 
-- [x] make a flowchart representing our statistical analyses, 
+- [x] make a flowchart representing our statistical analyses,
+- [ ] sample rarefaction: the documentation for `vegan::rrarefy` states that there are better ways to normalize sample sizes.
+> Random rarefaction is sometimes used to remove the effects of different sample sizes. This is usually a bad idea: random rarefaction discards valid data, introduces random error and reduces the quality of the data (McMurdie & Holmes 2014). It is better to use normalizing transformations (‘decostand’ in ‘vegan’) possible with variance stabilization (‘decostand’ and ‘dispweight’ in ‘vegan’) and methods that are not sensitive to sample sizes.
```

Comment:
https://journals.asm.org/doi/full/10.1128/msphere.00354-23. Schloss, 2024 state that rarefaction is the most effective way to correct for sampling effort. He point the difference of rarefing (subsamplign one time) and rarefaction (subsampling multiple time and make the average). There is to opposite view thus we try both.

Reply:
(we try both rarefaction and normalizing transformation)

Reply:
the pull request now includes the Schloss counterpoint.